internal/config/config.go

```diff
@@ -267,18 +266,9 @@ func New(configPath string, version string) *AppConfig {
 
 func (c *AppConfig) defaults() {
 	c.Config = &domain.Config{
-		Version: "dev",
-		Host:    "0.0.0.0",
-		Port:    42069,
-		Clients: map[string]*domain.Client{
```

Comment:
You should probably still initialize the map to not cause panics if it tries to lookup a key.

`Clients: map[string]*domain.Client{},`